awesome.c

```diff
@@ -402,8 +404,10 @@ static gint
 a_glib_poll(GPollFD *ufds, guint nfsd, gint timeout)
 {
     guint res;
-    struct timeval now, length_time;
-    float length;
+#ifndef NDEBUG
```

Comment:
No, that information is intended for the regular users, not the developers. If you get that warning, fix your config.

Reply:
Sorry, it was for my PC!